readme.md

```diff
@@ -625,6 +639,97 @@ Defaults[.setUser].first?.name //=> "Hello"
 Defaults[.dictionaryUser]["user"]?.name //=> "Hello"
 ```
 
+### Dynamic value
+
+There might be situations where you want to use `[String: Any]` directly.
+But `Defaults` need its value conforms to `Defaults.Serializable`, so here is a class `Defaults.AnySerializable` to overcome this limitation.
+
+`Defaults.AnySerializable` only available for `Value` which conforms to `Defaults.Serializable`.
+
+#### Primitive type
+
+`Defaults.AnySerializable` conforms to `ExpressibleByStringLiteral`, `ExpressibleByIntegerLiteral`, `ExpressibleByFloatLiteral`, `ExpressibleByBooleanLiteral`, `ExpressibleByNilLiteral`, `ExpressibleByArrayLiteral` and `ExpressibleByDictionaryLiteral`.
+
+So it can assign directly with these primitive types.
+
+```swift
+let any = Defaults.Key<Defaults.AnySerializable>("anyKey", default: 1)
+Defaults[any] = "🦄"
+```
+
+#### Other types
+
+##### Using `get`, `set`
+
+For other types you will have to assign it like this.
+
+```swift
+enum mime: String, Defaults.Serializable {
```

Comment:
Shouldn't the example use a type that it's not already possible to use without type erasure?

Reply:
Every type should be able to use even without the type erasure.
`Defaults.AnySerializable` should only be used in the wrapped types, ex. `Array`, `Set`, `Dictionary`.